site/app/Resources/views/default/project_search.html.twig

```diff
@@ -115,31 +111,14 @@
       <hr/>
     {% endverbatim %}
   </script>
-  {% endblock %}
+  {# passing algolia credentials to search.js #}
+  <div class="hidden" id="search-data" data-search='{ "appId": "{{ algolia_app_id }}", "apiKey": "{{ algolia_search_key }}", "searchPrefix": "{{ algolia_search_prefix }}", "indexName": "{{ searchType }}" }'></div>
```

Comment:
Please initialize these values from a JS file as shown in the Algolia documentation. https://www.algolia.com/doc/api-client/getting-started/instantiate-client-index/

To get the necessary parameters from the Ansible YAML-files into a JS file you can instruct Ansible to fill out a template. I can do that for you, but please

- Create a JS file where you set these parameters (only)
- Hardcode these parameters for now, and I'll then take care of generating this file with Ansible during build-time

Reply:
The other option is to generate a JS file through a Symfony tool at build time, as proposed for the categories. (See the other review comment.)